11-payment-encoding.md

```diff
@@ -24,7 +24,8 @@ encoding.
 
 A writer MUST encode the the payment request in Bech32 as specified in
 BIP-0173.  A reader MUST parse the address as Bech32 as specified in
-BIP-0173, and MUST fail if the checksum is incorrect.
+BIP-0173, and MUST fail if the checksum is incorrect. Note that we
+allow the address strings to be longer than 90 characters.
```

Comment:
I think this should be more than a "note", since it's actually (as you point out) a contradiction to the previous sentence.

How about:

    A writer MUST encode the the payment request in Bech32 as specified in
    BIP-0173, with the exception that the bech32 string MAY be longer than the 90 characters specified there.  A reader MUST parse the address as Bech32 as specified in
    BIP-0173 (also without the character limit), and MUST fail if the checksum is incorrect.

Reply:
Agreed!